cves/CVE-2017-5648.yml

```diff
@@ -121,9 +131,12 @@ discovered:
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
   answer:
-  date:
-  automated:
-  contest:
+    The vulnerability was discovered in tomcat, and reported by Brett Kail in
```

Comment:
Mention what contest

Reply:
There was not contest mention in the here, I was look at the wrong file before.